lib/proxy/ip-forward.php

```diff
@@ -152,15 +152,15 @@ function set_remote_address( $ip ) {
 /**
  * Return the defined verification key for a site
  *
- * @return string|int The verification key if available, or a random integer if no key is configured.
+ * @return string The verification key if available, or a string of random numbers if no key is configured.
  */
 function get_proxy_verification_key() {
 	if ( defined( 'WPCOM_VIP_PROXY_VERIFICATION' ) && ! empty( WPCOM_VIP_PROXY_VERIFICATION ) ) {
 		return WPCOM_VIP_PROXY_VERIFICATION;
```

Comment:
Just to be safe, we should probably cast this as a string as well and add a test for it.